modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -96,48 +88,37 @@ sudo apt install redis-server php-redis
 
 The installer will automatically launch Redis and configure it to launch at startup.
 
-After that, assuming that you don’t encounter any errors, restart Apache
-and the extension is ready to use.
+After that, assuming that you don’t encounter any errors, restart Apache and the extension is ready to use.
 
 ==== Additional notes for Redis vs. APCu on Memory Caching
 
-APCu is faster at local caching than Redis. If you have enough memory,
-use APCu for memory caching and Redis for file locking. If you are low
-on memory, use Redis for both.
+APCu is faster at local caching than Redis. If you have enough memory, use APCu for memory caching and Redis for file locking. If you are low on memory, use Redis for both.
 
 
 ==== Clearing the Redis Cache
 
-The Redis cache can be flushed from the command-line using the
-{rediscli_url}[redis-cli tool], as in the following example:
+The Redis cache can be flushed from the command-line using the {rediscli_url}[redis-cli tool], as in the following example:
 
 ----
 sudo redis-cli
 SELECT <dbIndex>
 FLUSHDB
 ----
 
-`<dbIndex>` is the number of Redis database where the cache is stored.
-It is zero by default at ownCloud. To check what yours is currently set
-to, check the `dbindex` value in `config/config.php`. Here’s an example
-of what to look for:
+`<dbIndex>` is the number of Redis database where the cache is stored. It is zero by default at ownCloud. To check what yours is currently set to, check the `dbindex` value in `config/config.php`. Here’s an example of what to look for:
```

Comment:
Where is the example?

Reply:
Thanks for spotting it 👍 
These things always happen on a bigger review of existing documents...

Reply:
fixed